server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -53,7 +53,6 @@ static TransportVersion def(int id) {
     @UpdateForV9(owner = UpdateForV9.Owner.CORE_INFRA) // remove the transport versions with which v9 will not need to interact
     public static final TransportVersion ZERO = def(0);
     public static final TransportVersion V_7_0_0 = def(7_00_00_99);
```

Comment:
Are you keeping 7.0 here?

Reply:
There's still some references to this in various tests